counsel.el

```diff
@@ -3196,9 +3199,9 @@ Example input with inclusion and exclusion file patterns:
   (interactive)
   (let ((counsel-ag-base-command
          (if (listp counsel-rg-base-command)
-             (append counsel-rg-base-command (counsel--rg-targets))
+             (append counsel-rg-base-command (funcall counsel-rg-target-function))
            (concat counsel-rg-base-command " "
-                   (mapconcat #'shell-quote-argument (counsel--rg-targets) " "))))
+                   (mapconcat #'shell-quote-argument (funcall counsel-rg-target-function) " "))))
```

Comment:
Nit: Please split these already long lines so that they fit in ideally 80, but at most 90, columns.